toolkit/tools/internal/buildpipeline/buildpipeline.go

```diff
@@ -222,5 +225,13 @@ func CleanupDockerChroot(chroot string) (err error) {
 		}
 	}
 
+	// create some folder(s) once chroot has been cleaned up
```

Comment:
Could you extend the comment to explain why we want to create new folders during clean-up? I'm actually curious myself.